apps/api/src/services/chat-export.service.ts

```
@@ -23,14 +23,19 @@ export interface ChatExportData {
-  /^\[(\d{1,2}\/\d{1,2}\/\d{2,4}),\s(\d{1,2}:\d{2}(?::\d{2})?)\s?(AM|PM)?\]\s(.+?):\s(.+)$/i;
-
-// Alternative format for system messages
-const SYSTEM_MESSAGE_REGEX =
-  /^\[(\d{1,2}\/\d{1,2}\/\d{2,4}),\s(\d{1,2}:\d{2}(?::\d{2})?)\s?(AM|PM)?\]\s(.+)$/i;
+// WhatsApp exports use two incompatible on-device formats depending on OS:
+//   iOS:     [DD/MM/YYYY, HH:MM:SS] Sender: Message
+//   Android: DD/MM/YYYY, HH:MM - Sender: Message  (no brackets, dash separator)
+// WhatsApp also sometimes inserts a narrow no-break space (U+202F) instead of
+// a regular space before AM/PM. Support both formats and both space kinds.
+const IOS_MESSAGE_REGEX =
+  /^\[(?<date>\d{1,2}\/\d{1,2}\/\d{2,4}),\s(?<time>\d{1,2}:\d{2}(?::\d{2})?)[\s ]?(?<meridiem>AM|PM)?\]\s(?<sender>.+?):\s(?<content>.+)$/i;
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)